src/GrpcProtos/capture.proto

```diff
@@ -155,7 +155,7 @@ message ApiScopeStart {
   fixed64 encoded_name_6 = 9;
   fixed64 encoded_name_7 = 10;
   fixed64 encoded_name_8 = 11;
-  bytes encoded_name_additional = 12;
+  repeated fixed64 encoded_name_additional = 12;
```

Comment:
Maybe expand the comment above explaining how the `encoded_name_additional` is also a sequence of `fixed64` instead of a `bytes` so that it can be allocated on an `Arena` (`std::string`s cannot be allocated there).

Reply:
Done